tests/drawer/test_draw_mpl.py

```diff
@@ -69,17 +69,19 @@ def test_standard_use():
 
 
 @pytest.mark.parametrize(
-    "strategy, initial_strategy, n_patches", [("gradient", "device", 4), ("device", "gradient", 9)]
+    "strategy, initial_strategy, n_lines", [("gradient", "device", 3), ("device", "gradient", 13)]
 )
-def test_expansion_strategy(strategy, initial_strategy, n_patches):
-    @qml.qnode(qml.device("default.qubit", wires=range(2)), expansion_strategy=initial_strategy)
+def test_expansion_strategy(strategy, initial_strategy, n_lines):
+    """Test that the expansion strategy keyword controls what operations are drawn."""
+
+    @qml.qnode(qml.device("default.qubit", wires=3), expansion_strategy=initial_strategy)
     def circuit():
-        qml.GroverOperator(wires=range(2))
```

Comment:
was this changed because we no longer decompose Grover?

Reply:
Yep.  We needed an Operator that would decompose differently based on the expansion_strategy keyword argument.